Data/TeamConfiguration.test.ts

```diff
@@ -0,0 +1,140 @@
+﻿import { JIRA_DOMAIN, JIRA_ORGANIZATION_ID, JIRA_SITE_ID } from "../lib/Constants";
+import { JiraClient } from "../lib/JiraClient";
+import { LogTester } from "../tests/LogTester";
+import { EngineeringExperienceSquad, TeamConfigurationData } from "./TeamConfiguration";
+import { fail } from 'assert';
+import { Team } from '../lib/Team';
+
+let jira: JiraClient;
+
+// All teams that exist in Jira, but do not create PRs and do not need boardId configured:
+const ignoredTeams = [
```

Comment:
not sure how to review that this list is correct either.

Reply:
If you remove an item, UT will fail.
If there's additional item, there's no harm.